<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inscription</class>
 <widget class="QWidget" name="inscription">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>441</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="loginLabel">
      <property name="text">
       <string>Son identifiant</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="LoginEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lblNom">
      <property name="text">
       <string>Son nom</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="nomEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="lblMdp">
      <property name="text">
       <string>Son mot de passe</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="MdpEdit">
      <property name="inputMethodHints">
       <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="lbltype">
      <property name="text">
       <string>Quel type d'utilisateur ?</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QComboBox" name="selectType"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="clubLbl">
      <property name="text">
       <string>Son Club ?</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QComboBox" name="selectClub"/>
    </item>
    <item row="12" column="0" colspan="2">
     <widget class="QPushButton" name="boutonInscrire">
      <property name="text">
       <string>Inscrire</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="indication">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>432</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>303</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Inscrire un nouvel utilisateur dans la base</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
